src/Rule.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import './Rule.css';
+
+function Rule({
```

Comment:
Why are you using a function component here? We are still handling class components...